### PR TITLE
deadline shortcut added to settings with type error

### PR DESCRIPTION
**Pull request by FadyFS**
## Changelog Description
We added to json files new keys to deadline so it has a proper interface with its relatives paths on different OS.

## Additional info
While opening the settings you will have a Attribute error, if u have any clue to solve it I'd like your help
## Testing notes:
1. Launch openpype locally
2. Open studio settings,
3. Go to Modules and look for Deadline

### CODE REVIEW

File: openpype/settings/defaults/system_settings/modules.json [hunk 1]
```diff
@@ -159,9 +159,16 @@
...
+        "enabled": false,
         "deadline_urls": {
-            "default": "http://127.0.0.1:8082"
+            "default":"http://deadline.prs.vfx.int:8082"
+        },
+        "deadline_paths": {
+            "default": {
+                "windows": "",
+                "darwin": "",
+                "linux": "/prod/softprod/apps/deadline/10.1.23.6/linux/deadlinemonitor10.desktop"
```
Comment (ClementHector) [inline openpype/settings/defaults/system_settings/modules.json:170]: replace the linux path by an empty string.

File: openpype/settings/defaults/system_settings/modules.json [hunk 2]
```diff
@@ -159,9 +159,16 @@
         "sites": {}
     },
     "deadline": {
-        "enabled": true,
+        "enabled": false,
         "deadline_urls": {
-            "default": "http://127.0.0.1:8082"
+            "default":"http://deadline.prs.vfx.int:8082"
```
Comment (ClementHector) [inline openpype/settings/defaults/system_settings/modules.json:164]: Put back the default value to default `"http://127.0.0.1:8082"`

File: openpype/settings/defaults/system_settings/modules.json [hunk 3]
```diff
@@ -159,9 +159,16 @@
         "sites": {}
     },
     "deadline": {
-        "enabled": true,
+        "enabled": false,
```
Comment (ClementHector) [inline openpype/settings/defaults/system_settings/modules.json:162]: Put back the value to true

**Comment by kalisp** [comment]
I don't understand what this should be doing. Just adding Deadline Monitor icon (btw path to the icon is hardcoded)?
Could you please elaborate on its use case?

**Comment by BigRoy** [comment]
This PR seems to try and expose a "launch deadline" from the OpenPype launcher?

The PR seems wrong in many ways - hardcoded paths, linux only, doesn't use application launching logic even though it tries to emulate the behavior somewhat with exposing `environment` etc.

I agree with @kalisp this PR is a bit unclear entirely why it's a needed feature - what does it offer and why do you need it like that? If it makes sense I suppose we can discuss cleaning up the code.

**Comment by kalisp** [comment]
Closing for now. Please provide a clear use case for this and reopen.

**Pull request closed without merging**